pages/storage_and_backup/object_storage/Versioning_Doc/guide.en-gb.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'Getting Started with Versioning Using OVHcloud APIs'
+excerpt: 'Learn how to enable and manage versioning for your OVHcloud Object Storage buckets using APIs. Future updates will cover the OVHcloud Control Panel.'
+updated: 2024-06-26
+---
+
+## Objective
+
+**This guide explains how to enable and manage versioning for your OVHcloud Object Storage buckets using APIs.**
+
+## Requirements
+
+- A Public Cloud project in your OVHcloud account
+- Access to the OVHcloud Control Panel
+- An S3 user already created
+- AWS CLI installed and configured
+
+## Instructions
+
+### Concept
+
+Versioning in Object Storage allows you to keep multiple variants of an object in the same bucket. This feature helps preserve, retrieve, and restore every version of every object stored in your buckets, making it easier to recover from unintended user actions or application failures. By default, versioning is disabled on buckets, and you must explicitly enable it.
+
+### General Information
+
+An Object Storage bucket can be in one of three states:
+
+1. **Unversioned** (default state): No versions are kept for the objects.
+2. **Versioning-enabled**: Multiple versions of each object are kept.
+3. **Versioning-suspended**: Versions are not created for new objects, but existing versions are retained.
+
+> [!warning]
+> Once versioning is enabled, it cannot be disabled; it can only be suspended.
+
+
+![Archive](images/versionning.png)
+
+Enabling and suspending versioning is done at the bucket level. Once enabled, all objects in the bucket will receive a unique version ID. Existing objects will have a version ID of null until they are modified.
+
+### How Versioning Works
+
+Versioning adds a layer of protection to your data by keeping multiple versions of an object in the same bucket. When you enable versioning for a bucket, every object in the bucket gets a unique version ID. This means that every time an object is modified or deleted, a new version is created, and the old version is retained. This allows you to recover previous versions of an object if necessary.
+
+- **Uploading new objects:** A unique version ID is assigned to each object.
+- **Modifying objects:** A new version ID is generated, and the previous version is retained.
+- **Deleting objects:** The deletion operation creates a delete marker but does not remove the previous versions. The object can be restored by removing the delete marker.
+
+### Version IDs
```

Comment:
Isn't this section supposed to be an H4 instead of H3? Same question for the following section.